content/800-data-platform/200-projects/100-create.mdx

```diff
@@ -0,0 +1,152 @@
+---
+title: 'Create projects'
+metaTitle: 'Create projects in the Prisma Data Platform'
+metaDescription: 'In the Prisma Data Platform, you can create projects from your JavaScript/TypeScript database applications that use Prisma. To do so, you need to have your database connection string at hand and configure access to the database from the Prisma Data Platform static IPs as well as select a Data Proxy region.'
+tocDepth: 1
+---
+
+<TopBlock>
+
+In the Prisma Data Platform, you can create new projects only with an available database connection string. If your database sits behind a firewall or only allows connections from specific IPs, you can configure the [static IPs of the platform](/data-platform/static-ips). You can also select the [Data Proxy](/data-platform/data-proxy) region to take advantage of connection pooling and load balancing in serverless environments.
+
+After you create your project, you can optionally link your Prisma schema (if you push it to a GitHub repository) so that we can keep your project in sync with its latest version and enable you and your team members to work with the [Data Browser](/data-platform/data-browser), [Query Console](/data-platform/query-console), and [Schema Viewer](/data-platform/schema-viewer).
+
+Later, if you want to enable connection pooling and load balancing of database connections with the Data Proxy, you can copy the Data Proxy connection string for the project.
+
+</TopBlock>
+
+## Create a project
+
+When you create a project, you only need a database connection string.
+
+### Prerequisites
+
+Copy the database connection string for your project. The platform uses the connection string only to read and introspect the database.
+
+<Admonition>
+
+**Note**<br /><br />
+
+* Use a [Prisma ORM-supported database](/reference/database-reference/supported-databases).
+* The connection string must include `user` and `password`
+* Use the connection string format below.
+    ```code no-copy
+    <db-type>://<user>:<password>@<hostname>:<port>/<db-name>
+    ```
+
+</Admonition>
+
+### Steps
+
+1. After you [sign up for the Prisma Data Platform](/data-platform/account#sign-up-for-the-prisma-data-platform), go to the [_Projects_](https://cloud.prisma.io/projects) page.
```

Comment:
After you sign up, you actually automatically get redirected to project creation